### PR TITLE
Add new paramter for azure_rm_aks module

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Add new paramter for azure_rm_aks module.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

azure_rm_aks
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by Fred-sun** [comment]
ready_for_review

File: plugins/modules/azure_rm_aks.py [hunk 1]
```diff
@@ -630,10 +691,28 @@ def compare_addon(origin, patch, config):
...
+                                elif profile_result['vm_size'] != profile_self['vm_size'] and profile_self['vm_size'] :
+                                    self.log(("Agent Profile Diff - Origin {0} / Update {1}".format(str(profile_result), str(profile_self))))
+                                    to_be_updated = True
+                                elif profile_result['os_disk_size_gb'] != os_disk_size_gb and os_disk_size_gb :
+                                    self.log(("Agent Profile Diff - Origin {0} / Update {1}".format(str(profile_result), str(profile_self))))
+                                    to_be_updated = True
+                                elif profile_result['vnet_subnet_id'] != profile_self.get('vnet_subnet_id', profile_result['vnet_subnet_id']) and profile_self.get('vnet_subnet_id', profile_result['vnet_subnet_id']) :
+                                    self.log(("Agent Profile Diff - Origin {0} / Update {1}".format(str(profile_result), str(profile_self))))
+                                    to_be_updated = True
+                                elif profile_result['type'] != profile_self['type'] and profile_self['type'] :
```
Comment (haiyuazhang) [inline plugins/modules/azure_rm_aks.py]: make sure type is updatable, otherwise we should remove it

Comment (Fred-sun) [reply plugins/modules/azure_rm_aks.py]: Thanks for your review! I will retest and check it!

**Pull request merged**